lib/convenient_service/support/dependency_container/entities/method_collection.rb

```diff
@@ -23,7 +23,7 @@ def find_by(name: Support::NOT_PASSED, full_name: Support::NOT_PASSED, scope: Su
             rules = []
 
             rules << ->(method) { method.name.to_s == name.to_s } if name != Support::NOT_PASSED
-            rules << ->(method) { method.full_name.to_s == full_name.to_s } if full_name != Support::NOT_PASSED
+            rules << ->(method) { method.slug.to_s == full_name.to_s } if full_name != Support::NOT_PASSED
```

Comment:
**Critical**

Please, rename `full_name` to `slug` in `MethodCollecttion` as well. Thanks.